akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -1124,6 +1126,8 @@ abstract class ShardCoordinator(
       }
     }
 
+  protected def unstashGetShardHomeRequests(): Unit
```

Comment:
This is the case in which I think we should unstash all.

https://github.com/akka/akka/blob/3a03fd5f39a37fe7a5312af921e5ad15fa928bd4/akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala#L1465-L1469

Reply:
It will be same for that case. Better to unstash one and then continue unstash next after it has been handled, because those will likely also result in updates.

Reply:
rename to `unstashOneGetShardHomeRequest`  sounds good